roles/kubernetes/node/defaults/main.yml

```diff
@@ -258,4 +261,4 @@ conntrack_modules:
 ## Enable distributed tracing for kubelet
 kubelet_tracing: false
 kubelet_tracing_endpoint: 0.0.0.0:4317
-kubelet_tracing_sampling_rate_per_million: 100
\ No newline at end of file
+kubelet_tracing_sampling_rate_per_million: 100
```

Comment:
HI @KubeKyrie 

The line may not  needs to be changed :-)

Reply:
Yeah, I changed the file and the editor helped me add a new line at end of file. The variable is not  be changed.
And generally the end of the file has a new line.😀
@yankay